tests/components/openerz/test_sensor.py

```diff
@@ -0,0 +1,75 @@
+"""Tests for OpenERZ component."""
+from copy import deepcopy
+from unittest.mock import MagicMock, patch
+
+from homeassistant.components.sensor import DOMAIN as SENSOR_DOMAIN
+from homeassistant.setup import async_setup_component
+
+MOCK_CONFIG = {
+    "sensor": {
+        "platform": "openerz",
+        "name": "test_name",
+        "zip": 1234,
+        "waste_type": "glass",
+    }
+}
+
+
+async def test_sensor_init(hass):
+    """Test whether all values initialized properly."""
+    with patch(
+        "homeassistant.components.openerz.sensor.OpenERZSensor.update",
```

Comment:
Patch the library method instead. We don't want to patch our integration code. That's what we want to test.